src/main/java/org/candlepin/model/ContentCurator.java

```diff
@@ -717,4 +717,45 @@ WITH RECURSIVE pool_products (product_uuid) AS (
         return output;
     }
 
+    /**
+     * Fetches the required product IDs for the contents specified by the collection of content UUIDs.
+     * The required product IDs will be returned as a mapping of consumer UUID to required product ID
```

Comment:
Update `mapping of consumer UUID` to `mapping of content UUID`

Reply:
You wouldn't believe the number of times I've made this mistake working with content. Fixed.